assets/templates/header-component.html

```diff
@@ -152,18 +152,178 @@
                         ><span class="email">gbaamonde@makingsense.com</span>
                       </p>
                     </header>
-                    <div class="user-plan--container m-t-18">
+                    <!-- <div class="user-plan--container m-t-18">
```

Comment:
Why do you have all this markup commented? cannot it be left in another page with all examples working?

Reply:
changed